### PR TITLE
filter out genres with empty labels

### PR DESCRIPTION
**Pull request by georgiaewhitney**
For #2386 

Genres with empty labels are filtered out

### CODE REVIEW

File: pipeline/transformer/transformer_sierra/src/main/scala/weco/pipeline/transformer/sierra/transformers/SierraGenres.scala [hunk 1]
```diff
@@ -48,18 +48,21 @@ object SierraGenres
...
-            primarySubfields,
-            varField = varField
-          ) ++ getSubdivisions(subdivisionSubfields)
+          getLabel(primarySubfields, subdivisionSubfields) match {
+            case "" => None
+            case label =>
+              val concepts = getPrimaryConcept(
+                primarySubfields,
+                varField = varField
+              ) ++ getSubdivisions(subdivisionSubfields)
```
Comment (alexwlchan) [inline pipeline/transformer/transformer_sierra/src/main/scala/weco/pipeline/transformer/sierra/transformers/SierraGenres.scala:62]: nit: I would keep the two variable assignments from the original code, then add the `match` block as a new thing – there's a lot going on in a couple of lines that make it a bit tricky to follow.

**Review by alexwlchan — APPROVED** [review]
One readability suggestion, not blocking.

File: pipeline/transformer/transformer_sierra/src/main/scala/weco/pipeline/transformer/sierra/transformers/SierraGenres.scala (hunk 1 above)
Comment (georgiaewhitney) [reply pipeline/transformer/transformer_sierra/src/main/scala/weco/pipeline/transformer/sierra/transformers/SierraGenres.scala:62]: Thanks, Alex! This is such a great learning point. 

Something like this?

```scala
val (primarySubfields, subdivisionSubfields) = getLabelSubfields(varField)
      val label = getLabel(primarySubfields, subdivisionSubfields)
      label match {
        case "" => None
        case _ =>
          val concepts = getPrimaryConcept(
            primarySubfields,
            varField = varField
          ) ++ getSubdivisions(subdivisionSubfields)
```

Comment (alexwlchan) [reply pipeline/transformer/transformer_sierra/src/main/scala/weco/pipeline/transformer/sierra/transformers/SierraGenres.scala:62]: I'd move the `concepts` variable out as well, so the `match` arms are both quite short. You'll only use the variable in one of the arms, but then you can glance over the `match` to see what each arm is doing.

**Pull request merged**